src/main/webapp/css/search-bar.css

```diff
@@ -212,6 +216,6 @@
 
 @media screen and (max-width: 560px) {
 	.search-wrapper.active .input-holder {
-		width: 200px;
+		width: 175px;
```

Comment:
should we make this relative to screen size?

Reply:
This style is only activated on small screens.

Reply:
ohh, got it!